app/views/posts/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<% @posts.each do |post| %>
+  <div class="well">
+    <h3><%= post.title %></h3>
+    <p><%= post.body %></p>
+    <%= link_to t("read.more"), post_path(post), class: "btn btn-default" %>
+  </div>
```

Comment:
You must create file '_post.html.erb', then move code lines to that file, then call back here